### PR TITLE
Project List Screen: plus button with a drop-down on client sections

### PR DESCRIPTION
**Pull request by masterrr**
screenshot_2015-03-30-16-25-26

Closes https://github.com/toggl/mobile/issues/576

### CODE REVIEW

**Comment by masterrr** [comment]
There will be no “plus” button for the group with no project in a next PR.

File: Joey/UI/Adapters/ProjectsRecyclerAdapter.cs [hunk 1]
```diff
@@ -13,6 +14,8 @@
 using Android.Support.V7.App;
 using Android.Support.V7.Widget;
```
Comment (monday8am) [inline Joey/UI/Adapters/ProjectsRecyclerAdapter.cs]: Organise imports in every file you touch.

**Comment by taavirehemagi** [comment]
Looks good to me

**Comment by masterrr** [comment]
We don't need this functionality for now.

**Pull request closed without merging**